stl/inc/span

```diff
@@ -415,7 +415,7 @@ public:
     constexpr span(const array<value_type, _Size>& _Arr) noexcept : _Mybase(_Size), _Mydata(_Arr.data()) {}
 
     template <class _Rng>
-        requires (_Extent == dynamic_extent) && _Is_span_compatible_range<_Rng, element_type>
+        requires _Is_span_compatible_range<_Rng, element_type>

```

Comment:
Note to reviewers (and myself), this is the concept so no `_v`

Reply:
Then shouldn't that drop the "is" to be consistent with the standard concepts which do the same?

Reply:
I am totally in favor of that. 

That said, we have currently 2 implementations, one for concept enabled compilers and one for those who do not. Until we can drop the non concept code with the adoption of clang-10 I would prefer to keep them both with the identical name, which makes me nearly wanna remove the `_v` again.

So I would propose to postpone the renaming until after clang-10 is required.